google-cloud-storage/src/main/java/com/google/cloud/storage/multipartupload/model/UploadPartRequest.java

```diff
@@ -85,6 +88,18 @@ public String uploadId() {
     return uploadId;
   }
 
+  /**
+   * Returns the CRC32C checksum of the part to upload.
+   *
+   * @return The CRC32C checksum of the part to upload.
+   * @since 2.60.0 This new api is in preview and is subject to breaking changes.
```

Comment:
```suggestion
   * @since 2.61.0 This new api is in preview and is subject to breaking changes.
```

v2.60.0 has already been released, the next feature release will be 2.61.0